taipy/gui/gui_actions.py

```diff
@@ -440,3 +440,17 @@ def thread_status(name: str, period_s: float, count: int):
     thread.start()
     if isinstance(period, int) and period >= 500 and callable(user_status_function):
         thread_status(thread.name, period / 1000.0, 0)
+
+
+def get_local_storage(state: State, *keys: str) -> t.Optional[t.Union[str, t.Dict[str, str]]]:
+    """Get local storage value(s).
+    Arguments:
```

Comment:
Line skip here.

Can this be called anytime in the app life cycle?

I suspect we should add more details on invoking this API (with the complexity of the keys type).

Reply:
i have not tested every case but it is very early on, only after the state id is initialized.